### PR TITLE
QAART-843

### PR DESCRIPTION
**Pull request by nikodamn**
After deleting avatar user needs to redirect to any page to get refreshed avatar.

Reviewers:
@ludwikkazmierczak @JurasJuras @justnpT

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/userprofile/UserAvatar.java [hunk 1]
```diff
@@ -59,20 +58,21 @@ public void clickOnAvatarRedirectsStaffUserToUserPage() {
...
     UserProfilePageObject profile = new UserProfilePageObject(driver).openProfilePage(
         credentials.userNameStaff, wikiURL);
     String avatarUrl = profile.getAvatarImageSrc();
     profile.clickRemoveAvatar();
     profile.verifyAvatar();
 
-    profile.verifyAvatarChanged(avatarUrl);
-    String changedAvatarUrl = profile.getAvatarImageSrc();
-    profile.verifyAvatarVisible();
+    profile.openWikiPage(); //user needs to visit other page to get avatar refreshed
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/userprofile/UserAvatar.java]: MINOR Move this trailing comment on the previous empty line. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**